incubator/aws-alb-ingress-controller/templates/clusterrole.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
The RBAC v1 API was available in 1.9, the same release the `apps/v1` API used elsewhere in the chart. Can you please update to the release version of the RBAC API.

Reply:
thanks so much Matt! I'll update them now and test it 😄